src/assets/styles/global.scss

```diff
@@ -2,51 +2,31 @@
 @import "./node_modules/@esri/calcite-base/dist/_index.scss";
 
 /* CSS vars (@include in global) */
-@import "layout";
-@import "shadow";
+@import "type";
 
 :root {
-  font-family: $avenir-family;
+  @extend %type-vars;
+  @include calcite-theme-light(); 
+
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-
-  @include calcite-theme-light();
-  @include componentLayoutVars();
-  @include componentShadowVars();
-
-  --calcite-border-radius: 3px;
-  --calcite-icon-size: 1rem;
-  --calcite-ui-opacity-disabled: 0.5;
-
-  --calcite-ui-text-weight-light: 300;
-  --calcite-ui-text-weight: 400;
-  --calcite-ui-text-weight-medium: 500;
-  --calcite-ui-text-weight-demi: 700;
-
-  --calcite-code-family: #{$code-family};
-
-  --calcite-ui-foreground-current: #{$h-bb-010};
-  --calcite-ui-text-current: #{$h-bb-070};
+  
+  @apply font-sans;
 }
 
 [theme="dark"] {
   @include calcite-theme-dark();
-  --calcite-ui-foreground-current: #{$blk-220};
-  --calcite-ui-text-current: #{$v-bb-120};
 }
 
 [theme="light"] {
   @include calcite-theme-light();
-  --calcite-ui-foreground-current: #{$h-bb-010};
-  --calcite-ui-text-current: #{$h-bb-070};
 }
 
 .overflow-hidden {
-  overflow: hidden;
+  @apply overflow-hidden;
 }
 
-[calcite-hydrated-hidden] {
-  visibility: hidden;
-  pointer-events: none;
+[calcite-hydrated][aria-hidden="true"] {
```

Comment:
I think this may be incorrect. Should use the deleted selector (but probably your `@apply` rules. This may fix the tests as well.

Reply:
Thanks @paulcpederson! I'll has the looksee.

Reply:
@driskull Do you know how this might affect CalciteIcon and CalciteDropdown?
I reverted locally to not use `@apply`, and am getting the same errors. Thanks!

Reply:
@asangma I don't think these have anything to do with those two components.

It should remain as `[calcite-hydrated-hidden]` though.

Reply:
Thanks @driskull. I'll try that.

Reply:
Thanks @driskull.
With some screenshot updates and some tweaks to reset soon-to-be-refactored components that were triggering Screener, the build seems to be good now.